File: 3232-find-if-digit-game-can-be-won/3232-find-if-digit-game-can-be-won.py
class Solution(object):
    def canAliceWin(self, nums):
        sum=0
        sum2=0
        for n in range(len(nums)):
            if nums[n]<10:
                sum+=nums[n]
            else:
                sum2+=nums[n]
        if sum==sum2:
            return False
        else :
            return True

        

File: 3232-find-if-digit-game-can-be-won/3232-find-if-digit-game-can-be-won.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var canAliceWin = function(nums) {
    let sum=0
    let sum2=0
    for (i=0;i<nums.length;i++){
        if(nums[i]<10){
            sum+=nums[i]
        }else{
            sum2+=nums[i]
        }
    }
    if(sum==sum2){
        return false;
    }else{
        return true;
    }
};